src/Controls/tests/DeviceTests/Resources/Raw/HybridTestRoot/index.html

```diff
@@ -93,6 +93,38 @@
             return response.status;
         }
 
+        async function RequestsWithAppUriCanBeIntercepted() {
+            const response = await fetch(`${window.location.origin}/api/sample?param1=value1&param2=value2`);
+            const jsonData = await response.json();
+            return jsonData;
+        }
+
+        async function RequestsCanBeIntercepted() {
+            const response = await fetch('https://echo.free.beeceptor.com/sample-request?param1=value1&param2=value2', {
```

Comment:
is this the best URL we can use? can we point to maybe something on our repository? 

IF the url goes away the test will fail.